### PR TITLE
YLP-478  Check gatekeeper usage in platform service

### PR DESCRIPTION
**Pull request by CedricPerotto**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                    }
+                }
+            }
+        }
+        stage('Test ') {
+            steps {
+                echo 'start mongo to serve as a testing db'
+                sh """
+                    docker network create platform_build
+                    docker container run -d --ulimit nofile=1048576 --name mongo4platform --network=platform_build mongo:4.2
```
Comment (ydrisR) [inline Jenkinsfile]:
```suggestion
                    docker container run -d --ulimit nofile=1048576 --name mongo4platform${BUILD_NUMBER} --network=platform_build${BUILD_NUMBER} mongo:4.2

```

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                """
+                script {
+                    builderImage.inside("--network=platform_build") {
```
Comment (ydrisR) [inline Jenkinsfile]:
```suggestion
                    builderImage.inside("--network=platform_build${BUILD_NUMBER}") {

```

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                        sh "make ci-generate ci-build"
+                    }
+                }
+            }
+        }
+        stage('Test ') {
+            steps {
+                echo 'start mongo to serve as a testing db'
+                sh """
+                    docker network create platform_build
```
Comment (ydrisR) [inline Jenkinsfile]:
```suggestion
                    docker network create platform_build${BUILD_NUMBER}

```

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                script {
+                    builderImage.inside("--network=platform_build") {
+                        sh "JENKINS_TEST=on make ci-test"
+                    }
+                }
+            }
+            post {
+                always {
+                    sh """
+                        docker logs mongo4platform > mongo4platform.log
```
Comment (ydrisR) [inline Jenkinsfile]:
```suggestion
                        docker logs mongo4platform${BUILD_NUMBER} > mongo4platform.log

```

File: Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                        gzip -9f mongo4platform.log
+                    """
+                    archiveArtifacts artifacts: 'mongo4platform.log.gz'
+                    sh 'docker stop mongo4platform && docker rm mongo4platform  && docker network rm platform_build'
```
Comment (ydrisR) [inline Jenkinsfile]:
```suggestion
                    sh 'docker stop mongo4platform${BUILD_NUMBER} && docker rm mongo4platform${BUILD_NUMBER}  && docker network rm platform_build${BUILD_NUMBER}'

```

**Review by ydrisR — CHANGES_REQUESTED** [review]
Just a few suggestions on Jenkinsfile to avoid conflicts when 2 builds run in //
And one question: can't we reuse go common coastguard client lib?

**Review by ydrisR — APPROVED** [review]
LGTM

**Comment by CedricPerotto** [comment]
> And one question: can't we reuse go common coastguard client lib?

We could but until now platform didn't use go-common at all...

**Pull request merged**